frontend/src/components/header/HeaderComponent.js

```diff
@@ -91,10 +115,12 @@ function HeaderComponent() {
         case currentItem === SLUGS.industrialApplications:
             title = 'Industrial Applications';
         case currentItem === SLUGS.details:
-            title = 'Details: ' + (selectedProducts[0].productName === undefined ? 'Please select a model first' : selectedProducts[0].productName);
+            title='Details '
+            subtitle = ' Selected product  ' ;
+            subsubtitle =(selectedProducts[0].productName === undefined ? 'Please select a model first' : selectedProducts[0].productName);
```

Comment:
yes because I commented it out in the main function. That is the reason why It is not showing. I am not sure if we are going to add these on the header as well or not . Thats why. Lets also discuss over it :)

Reply:
I  added that on and pushed it . Please do "git pull" as well. Now you can see the product filter on the header bar on the details page as well :) @Waldleufer